Sources/Controllers/ALKConversationViewController.swift

```diff
@@ -2443,6 +2443,13 @@ extension ALKConversationViewController: ALMQTTConversationDelegate {
               UIApplication.shared.applicationState != .background else { return false }
         return true
     }
+    
+    public func handOff(toHuman conversationId: String!) {
```

Comment:
Add Zendesk/Zopim suffix to separate it from other methods.

Reply:
updated the name as `handOffToZendeskAgent`